### DISCUSSION TITLE
OpenDTU goes offline every second or third day

### DISCUSSION THREAD

**Discussion by robson1311** [tbnobody/OpenDTU#2169] [Q&A]
I have been using OpenDTU and a storage system in conjunction with the [HoymilesZeroExport](https://github.com/reserve85/HoymilesZeroExport) script by reserve85 for quite some time.
It has always worked perfectly, but recently, OpenDTU stops operating every 2-3 days, causing the system to halt. 
The HM-1200 then continues to feed in the last used wattage and is no longer regulated. To reactivate the system, I have to restart OpenDTU, after which everything is regulated again until it goes offline once more. The system is fully up to date, and various updates have not resolved the issue.
I am attaching the last console log file before the restart. I hope someone can help! 
Thank` you very much!

Websocket: [/livedata][31] disconnect
21:32:23.507 > RX Period End
21:32:23.508 > All missing
21:32:23.508 > Nothing received, resend whole request
21:32:23.508 > TX ActivePowerControl Channel: 3 --> 51 71 60 56 25 80 18 48 04 81 0B 00 00 E6 00 01 57 21 FC 
21:32:25.543 > RX Period End
21:32:25.543 > All missing
21:32:25.543 > Nothing received, resend whole request
21:32:25.543 > TX ActivePowerControl Channel: 23 --> 51 71 60 56 25 80 18 48 04 81 0B 00 00 E6 00 01 57 21 FC 
21:32:27.574 > RX Period End
21:32:27.574 > All missing
21:32:27.574 > Nothing received, resend whole request
21:32:27.574 > TX ActivePowerControl Channel: 40 --> 51 71 60 56 25 80 18 48 04 81 0B 00 00 E6 00 01 57 21 FC 
21:32:29.614 > RX Period End
21:32:29.614 > All missing
21:32:29.614 > Nothing received, resend count exeeded
21:32:29.614 > Fetch inverter: 116171605625
21:32:29.614 > Request SystemConfigPara
21:32:29.614 > Resend ActivePowerControl
21:32:29.771 > TX RealTimeRunData Channel: 61 --> 15 71 60 56 25 80 18 48 04 80 0B 00 66 A1 56 CE 00 00 00 00 00 00 00 00 87 D0 20 
21:32:30.313 > RX Period End
21:32:30.313 > All missing
21:32:30.313 > Nothing received, resend whole request
21:32:30.313 > TX RealTimeRunData Channel: 75 --> 15 71 60 56 25 80 18 48 04 80 0B 00 66 A1 56 CE 00 00 00 00 00 00 00 00 87 D0 20 
21:32:30.842 > RX Period End
21:32:30.842 > All missing
21:32:30.842 > Nothing received, resend whole request
21:32:30.842 > TX RealTimeRunData Channel: 3 --> 15 71 60 56 25 80 18 48 04 80 0B 00 66 A1 56 CE 00 00 00 00 00 00 00 00 87 D0 20 
21:32:31.382 > RX Period End
21:32:31.382 > All missing
21:32:31.382 > Nothing received, resend whole request
21:32:31.382 > TX RealTimeRunData Channel: 23 --> 15 71 60 56 25 80 18 48 04 80 0B 00 66 A1 56 CE 00 00 00 00 00 00 00 00 87 D0 20 
21:32:31.917 > RX Period End
21:32:31.917 > All missing
21:32:31.917 > Nothing received, resend whole request
21:32:31.917 > TX RealTimeRunData Channel: 40 --> 15 71 60 56 25 80 18 48 04 80 0B 00 66 A1 56 CE 00 00 00 00 00 00 00 00 87 D0 20 
21:32:32.415 > RX Period End
21:32:32.415 > All missing
21:32:32.415 > Nothing received, resend count exeeded
21:32:32.474 > TX SystemConfigPara Channel: 61 --> 15 71 60 56 25 80 18 48 04 80 05 00 66 A1 56 CE 00 00 00 00 00 00 00 00 49 DF EF 
21:32:32.683 > RX Period End
21:32:32.683 > All missing
21:32:32.683 > Nothing received, resend whole request
21:32:32.683 > TX SystemConfigPara Channel: 75 --> 15 71 60 56 25 80 18 48 04 80 05 00 66 A1 56 CE 00 00 00 00 00 00 00 00 49 DF EF 
21:32:32.918 > RX Period End
21:32:32.918 > All missing
21:32:32.918 > Nothing received, resend whole request
21:32:32.918 > TX SystemConfigPara Channel: 3 --> 15 71 60 56 25 80 18 48 04 80 05 00 66 A1 56 CE 00 00 00 00 00 00 00 00 49 DF EF 
21:32:33.149 > RX Period End
21:32:33.149 > All missing
21:32:33.149 > Nothing received, resend whole request
21:32:33.149 > TX SystemConfigPara Channel: 23 --> 15 71 60 56 25 80 18 48 04 80 05 00 66 A1 56 CE 00 00 00 00 00 00 00 00 49 DF EF 
21:32:33.386 > RX Period End
21:32:33.386 > All missing
21:32:33.386 > Nothing received, resend whole request
21:32:33.386 > TX SystemConfigPara Channel: 40 --> 15 71 60 56 25 80 18 48 04 80 05 00 66 A1 56 CE 00 00 00 00 00 00 00 00 49 DF EF 
21:32:33.588 > RX Period End
21:32:33.588 > All missing
21:32:33.588 > Nothing received, resend count exeeded
21:32:33.639 > TX ActivePowerControl Channel: 61 --> 51 71 60 56 25 80 18 48 04 81 0B 00 00 E6 00 01 57 21 FC 
21:32:35.654 > RX Period End
21:32:35.654 > All missing
21:32:35.654 > Nothing received, resend whole request
21:32:35.654 > TX ActivePowerControl Channel: 75 --> 51 71 60 56 25 80 18 48 04 81 0B 00 00 E6 00 01 57 21 FC 
21:32:37.688 > RX Period End
21:32:37.688 > All missing
21:32:37.688 > Nothing received, resend whole request
21:32:37.688 > TX ActivePowerControl Channel: 3 --> 51 71 60 56 25 80 18 48 04 81 0B 00 00 E6 00 01 57 21 FC 
21:32:39.722 > RX Period End
21:32:39.722 > All missing
21:32:39.722 > Nothing received, resend whole request
21:32:39.722 > TX ActivePowerControl Channel: 23 --> 51 71 60 56 25 80 18 48 04 81 0B 00 00 E6 00 01 57 21 FC 
21:32:41.756 > RX Period End
21:32:41.756 > All missing
21:32:41.756 > Nothing received, resend whole request
21:32:41.756 > TX ActivePowerControl Channel: 40 --> 51 71 60 56 25 80 18 48 04 81 0B 00 00 E6 00 01 57 21 FC 
21:32:43.782 > RX Period End
21:32:43.782 > All missing

**Comment by robson1311** [comment]
**This is what it looks like after the restart!**

Admin AP remaining seconds: 150 / 180
22:16:11.611 > Fetch inverter: 116171605625
22:16:11.666 > TX RealTimeRunData Channel: 3 --> 15 71 60 56 25 80 18 48 04 80 0B 00 66 A1 61 0C 00 00 00 00 00 00 00 00 C7 28 6D 
22:16:12.219 > RX Period End
22:16:12.219 > All missing
22:16:12.219 > Nothing received, resend whole request
22:16:12.219 > TX RealTimeRunData Channel: 23 --> 15 71 60 56 25 80 18 48 04 80 0B 00 66 A1 61 0C 00 00 00 00 00 00 00 00 C7 28 6D 
22:16:12.763 > RX Period End
22:16:12.763 > All missing
22:16:12.763 > Nothing received, resend whole request
22:16:12.763 > TX RealTimeRunData Channel: 40 --> 15 71 60 56 25 80 18 48 04 80 0B 00 66 A1 61 0C 00 00 00 00 00 00 00 00 C7 28 6D 
22:16:12.814 > Interrupt received
22:16:12.883 > RX Channel: 23 --> 95 71 60 56 25 71 60 56 25 01 00 01 01 C4 00 D2 00 CB 03 B3 03 95 00 0C 38 CA 91 | -80 dBm
22:16:12.942 > Interrupt received
22:16:13.002 > RX Channel: 75 --> 95 71 60 56 25 71 60 56 25 02 00 0D 78 47 8C 54 87 D8 01 C5 00 CD 00 CD 03 9D 78 | -80 dBm
22:16:13.068 > Interrupt received
22:16:13.150 > RX Channel: 3 --> 95 71 60 56 25 71 60 56 25 03 03 A0 00 10 38 0F 00 0D 6A A3 87 75 87 F3 09 1D 44 | -80 dBm
22:16:13.296 > RX Period End
22:16:13.296 > Last missing
22:16:13.296 > Request retransmit: 4
22:16:13.296 > TX RequestFrame Channel: 61 --> 15 71 60 56 25 80 18 48 04 84 27 
22:16:13.355 > Interrupt received
22:16:13.474 > RX Channel: 23 --> 95 71 60 56 25 71 60 56 25 84 13 8A 0D DE 00 C0 00 98 03 E7 01 3C 00 17 E0 E9 C4 | -80 dBm
22:16:13.598 > RX Period End
22:16:13.598 > Success
22:16:16.614 > Fetch inverter: 116171605625
22:16:16.667 > TX RealTimeRunData Channel: 75 --> 15 71 60 56 25 80 18 48 04 80 0B 00 66 A1 61 11 00 00 00 00 00 00 00 00 97 BD B5 
22:16:16.717 > Interrupt received
22:16:16.768 > RX Channel: 23 --> 95 71 60 56 25 71 60 56 25 02 00 0D 78 47 8C 54 87 D8 01 C5 00 CC 00 CD 03 9B 7F | -80 dBm
22:16:16.816 > Interrupt received
22:16:16.867 > RX Channel: 40 --> 95 71 60 56 25 71 60 56 25 03 03 9E 00 10 38 0F 00 0D 6A A3 87 75 87 F3 09 1E 79 | -80 dBm
22:16:16.920 > Interrupt received
22:16:16.967 > RX Channel: 23 --> 95 71 60 56 25 71 60 56 25 84 13 8A 0D DE 00 C0 00 98 03 E7 01 3C 00 17 25 00 E8 | -80 dBm
22:16:17.262 > RX Period End
22:16:17.262 > Middle missing
22:16:17.262 > Request retransmit: 1
22:16:17.262 > TX RequestFrame Channel: 3 --> 15 71 60 56 25 80 18 48 04 81 22 
22:16:17.315 > Interrupt received
22:16:17.364 > RX Channel: 40 --> 95 71 60 56 25 71 60 56 25 01 00 01 01 C4 00 D2 00 CC 03 B5 03 96 00 0C 38 CA 93 | -80 dBm
22:16:17.675 > RX Period End
22:16:17.675 > Success
22:16:17.869 > TX ActivePowerControl Channel: 23 --> 51 71 60 56 25 80 18 48 04 81 0B 00 01 40 00 01 88 C0 65 
22:16:17.918 > Interrupt received
22:16:17.973 > RX Channel: 61 --> D1 71 60 56 25 71 60 56 25 81 00 00 0B 00 14 07 48 | -80 dBm
22:16:19.336 > RX Period End
22:16:19.336 > Success
22:16:21.619 > Fetch inverter: 116171605625
22:16:21.720 > Admin AP remaining seconds: 160 / 180
22:16:21.720 > TX RealTimeRunData Channel: 40 --> 15 71 60 56 25 80 18 48 04 80 0B 00 66 A1 61 16 00 00 00 00 00 00 00 00 A7 9B A4 
22:16:21.766 > Interrupt received
22:16:21.820 > RX Channel: 23 --> 95 71 60 56 25 71 60 56 25 01 00 01 01 C4 00 DA 00 D3 03 D7 03 B7 00 0C 38 CA C7 | -80 dBm
22:16:21.882 > Interrupt received
22:16:21.933 > RX Channel: 75 --> 95 71 60 56 25 71 60 56 25 02 00 0D 78 47 8C 54 87 D8 01 C5 00 D5 00 D6 03 C4 22 | -80 dBm
22:16:21.985 > Interrupt received
22:16:22.036 > RX Channel: 61 --> 95 71 60 56 25 71 60 56 25 03 03 C7 00 10 38 0F 00 0D 6A A4 87 75 87 F4 09 1F 21 | -80 dBm
22:16:22.085 > Interrupt received
22:16:22.137 > RX Channel: 61 --> 95 71 60 56 25 71 60 56 25 84 13 8A 0E 6A 00 C3 00 9E 03 E7 01 3C 00 17 DE 55 F4 | -80 dBm
22:16:22.200 > RX Period End
22:16:22.200 > Success
22:16:26.619 > Fetch inverter: 116171605625
22:16:26.673 > TX RealTimeRunData Channel: 61 --> 15 71 60 56 25 80 18 48 04 80 0B 00 66 A1 61 1B 00 00 00 00 00 00 00 00 37 C3 61 
22:16:26.772 > Interrupt received
22:16:26.873 > RX Channel: 23 --> 95 71 60 56 25 71 60 56 25 01 00 01 01 C4 00 E7 00 E0 04 13 03 F1 00 0C 38 CA 4C | -80 dBm
22:16:27.190 > Interrupt received
22:16:27.256 > RX Channel: 3 --> 95 71 60 56 25 71 60 56 25 02 00 0D 78 48 8C 54 87 D9 01 C4 00 E3 00 E4 03 FF 12 | -80 dBm
22:16:27.318 > Interrupt received
22:16:27.390 > RX Channel: 23 --> 95 71 60 56 25 71 60 56 25 03 04 02 00 10 38 0F 00 0D 6A A4 87 75 87 F4 09 1F E3 | -80 dBm
22:16:27.484 > Interrupt received
22:16:27.534 > RX Channel: 3 --> 95 71 60 56 25 71 60 56 25 84 13 8A 0F 4D 00 C3 00 A8 03 E7 01 3D 00 17 DA 90 24 | -80 dBm
22:16:27.593 > RX Period End
22:16:27.593 > Success
22:16:31.621 > Fetch inverter: 116171605625
22:16:31.684 > TX RealTimeRunData Channel: 75 --> 15 71 60 56 25 80 18 48 04 80 0B 00 66 A1 61 20 00 00 00 00 00 00 00 00 06 E4 4C 
22:16:31.733 > Interrupt received
22:16:31.788 > RX Channel: 61 --> 95 71 60 56 25 71 60 56 25 01 00 01 01 C4 00 E7 00 E0 04 12 03 F0 00 0C 38 CB 4D | -80 dBm
22:16:31.851 > Admin AP remaining seconds: 170 / 180
22:16:31.905 > Interrupt received
22:16:32.305 > RX Channel: 23 --> 95 71 60 56 25 71 60 56 25 02 00 0D 78 48 8C 55 87 D9 01 C4 00 E3 00 E4 04 00 EB | -80 dBm
22:16:32.507 > Interrupt received
22:16:32.606 > RX Channel: 23 --> 95 71 60 56 25 71 60 56 25 03 04 03 00 10 38 10 00 0D 6A A4 87 76 87 F4 09 1F FE | -80 dBm
22:16:32.708 > Interrupt received
22:16:32.915 > RX Channel: 23 --> 95 71 60 56 25 71 60 56 25 84 13 8A 0F 4D 00 C3 00 A8 03 E7 01 3D 00 17 E3 1C 91 | -80 dBm
22:16:32.969 > RX Period End
22:16:32.969 > Success
22:16:36.468 > TX ActivePowerControl Channel: 3 --> 51 71 60 56 25 80 18 48 04 81 0B 00 01 2C 00 01 95 00 D4 
22:16:36.530 > Interrupt received
22:16:36.590 > RX Channel: 61 --> D1 71 60 56 25 71 60 56 25 81 00 00 0B 00 14 07 48 | -80 dBm
22:16:38.480 > RX Period End
22:16:38.480 > Success
22:16:38.480 > Fetch inverter: 116171605625
22:16:38.548 > TX RealTimeRunData Channel: 23 --> 15 71 60 56 25 80 18 48 04 80 0B 00 66 A1 61 27 00 00 00 00 00 00 00 00 36 C2 5D 
22:16:39.089 > RX Period End
22:16:39.089 > All missing
22:16:39.089 > Nothing received, resend whole request
22:16:39.089 > TX RealTimeRunData Channel: 40 --> 15 71 60 56 25 80 18 48 04 80 0B 00 66 A1 61 27 00 00 00 00 00 00 00 00 36 C2 5D 
22:16:39.268 > Interrupt received
22:16:39.318 > RX Channel: 3 --> 95 71 60 56 25 71 60 56 25 01 00 01 01 C4 00 E7 00 DF 04 10 03 EE 00 0C 38 CB 6E | -80 dBm
22:16:39.369 > Interrupt received
22:16:39.417 > RX Channel: 23 --> 95 71 60 56 25 71 60 56 25 02 00 0D 78 48 8C 55 87 D9 01 C4 00 E3 00 E3 03 FE 15 | -80 dBm
22:16:39.468 > Interrupt received
22:16:39.516 > RX Channel: 75 --> 95 71 60 56 25 71 60 56 25 03 04 01 00 10 38 10 00 0D 6A A4 87 76 87 F4 09 20 C3 | -80 dBm
22:16:39.567 > Interrupt received
22:16:39.674 > RX Channel: 3 --> 95 71 60 56 25 71 60 56 25 84 13 89 0F 46 00 C0 00 A7 03 E7 01 3D 00 17 F6 FF 63 | -80 dBm
22:16:39.783 > RX Period End
22:16:39.783 > Success
22:16:41.971 > Admin AP remaining seconds: 180 / 180
22:16:42.974 > Admin mode disabled
22:16:43.472 > Fetch inverter: 116171605625
22:16:43.561 > TX RealTimeRunData Channel: 61 --> 15 71 60 56 25 80 18 48 04 80 0B 00 66 A1 61 2C 00 00 00 00 00 00 00 00 06 B1 15 
22:16:43.617 > Interrupt received
22:16:43.669 > RX Channel: 23 --> 95 71 60 56 25 71 60 56 25 01 00 01 01 C4 00 D9 00 D2 03 D1 03 B1 00 0C 38 CB C4 | -80 dBm
22:16:43.718 > Interrupt received
22:16:43.766 > RX Channel: 3 --> 95 71 60 56 25 71 60 56 25 02 00 0D 78 48 8C 55 87 D9 01 C5 00 D5 00 D5 03 C0 2A | -80 dBm
22:16:43.817 > Interrupt received
22:16:43.870 > RX Channel: 3 --> 95 71 60 56 25 71 60 56 25 03 03 C3 00 10 38 10 00 0D 6A A4 87 76 87 F4 09 1F 39 | -80 dBm
22:16:43.926 > Interrupt received
22:16:43.987 > RX Channel: 3 --> 95 71 60 56 25 71 60 56 25 84 13 89 0E 58 00 C0 00 9D 03 E7 01 3C 00 17 5C 2D 3F | -80 dBm
22:16:44.036 > RX Period End
22:16:44.036 > Success
22:16:48.478 > Fetch inverter: 116171605625
22:16:48.524 > TX RealTimeRunData Channel: 75 --> 15 71 60 56 25 80 18 48 04 80 0B 00 66 A1 61 31 00 00 00 00 00 00 00 00 56 24 CD 
22:16:48.581 > Interrupt received
22:16:48.636 > RX Channel: 40 --> 95 71 60 56 25 71 60 56 25 02 00 0D 78 48 8C 55 87 D9 01 C5 00 D5 00 D5 03 C0 2A | -80 dBm
22:16:48.689 > Interrupt received
22:16:48.784 > RX Channel: 23 --> 95 71 60 56 25 71 60 56 25 03 03 C3 00 10 38 10 00 0D 6A A4 87 76 87 F4 09 1E 38 | -80 dBm
22:16:48.836 > Interrupt received
22:16:48.886 > RX Channel: 40 --> 95 71 60 56 25 71 60 56 25 84 13 88 0E 58 00 C0 00 9D 03 E7 01 3D 00 17 1D BF EC | -80 dBm
22:16:49.051 > RX Period End
22:16:49.051 > Middle missing
22:16:49.051 > Request retransmit: 1
22:16:49.051 > TX RequestFrame Channel: 3 --> 15 71 60 56 25 80 18 48 04 81 22 
22:16:49.102 > Interrupt received
22:16:49.155 > RX Channel: 40 --> 95 71 60 56 25 71 60 56 25 01 00 01 01 C4 00 D9 00 D2 03 D1 03 B1 00 0C 38 CB C4 | -80 dBm
22:16:49.209 > RX Period End
22:16:49.209 > Success
22:16:53.489 > Fetch inverter: 116171605625
22:16:53.601 > TX RealTimeRunData Channel: 23 --> 15 71 60 56 25 80 18 48 04 80 0B 00 66 A1 61 36 00 00 00 00 00 00 00 00 66 02 DC 
22:16:54.078 > RX Period End
22:16:54.078 > All missing
22:16:54.078 > Nothing received, resend whole request
22:16:54.078 > TX RealTimeRunData Channel: 40 --> 15 71 60 56 25 80 18 48 04 80 0B 00 66 A1 61 36 00 00 00 00 00 00 00 00 66 02 DC 
22:16:54.215 > Interrupt received
22:16:54.271 > RX Channel: 3 --> 95 71 60 56 25 71 60 56 25 01 00 01 01 C4 00 D9 00 D2 03 D1 03 B1 00 0C 38 CB C4 | -80 dBm
22:16:54.318 > Interrupt received
22:16:54.382 > RX Channel: 3 --> 95 71 60 56 25 71 60 56 25 02 00 0D 78 48 8C 55 87 D9 01 C5 00 D5 00 D5 03 C0 2A | -80 dBm
22:16:54.443 > Interrupt received
22:16:54.521 > RX Channel: 75 --> 95 71 60 56 25 71 60 56 25 03 03 C3 00 10 38 10 00 0D 6A A4 87 76 87 F4 09 1D 3B | -80 dBm
22:16:54.571 > Interrupt received
22:16:54.679 > RX Channel: 61 --> 95 71 60 56 25 71 60 56 25 84 13 88 0E 58 00 C0 00 9D 03 E7 01 3D 00 17 1C FC AE | -80 dBm
22:16:54.792 > RX Period End
22:16:54.792 > Success
22:16:58.486 > Fetch inverter: 116171605625
22:16:58.546 > TX RealTimeRunData Channel: 61 --> 15 71 60 56 25 80 18 48 04 80 0B 00 66 A1 61 3B 00 00 00 00 00 00 00 00 F6 5A 19 
22:16:58.831 > Interrupt received
22:16:58.889 > RX Channel: 23 --> 95 71 60 56 25 71 60 56 25 01 00 01 01 C4 00 D9 00 D2 03 D1 03 B1 00 0C 38 CB C4 | -80 dBm
22:16:58.942 > Interrupt received
22:16:58.999 > RX Channel: 3 --> 95 71 60 56 25 71 60 56 25 02 00 0D 78 48 8C 55 87 D9 01 C5 00 D5 00 D5 03 C0 2A | -80 dBm
22:16:59.055 > Interrupt received
22:16:59.229 > RX Channel: 3 --> 95 71 60 56 25 71 60 56 25 03 03 C3 00 10 38 10 00 0D 6A A5 87 76 87 F5 09 1C 3A | -80 dBm
22:16:59.300 > Interrupt received
22:16:59.361 > RX Channel: 75 --> 95 71 60 56 25 71 60 56 25 84 13 88 0E 58 00 C0 00 9D 03 E7 01 3D 00 17 5D 7C 6F | -80 dBm
22:16:59.433 > RX Period End
22:16:59.433 > Success
22:17:03.491 > Fetch inverter: 116171605625
22:17:03.556 > TX RealTimeRunData Channel: 75 --> 15 71 60 56 25 80 18 48 04 80 0B 00 66 A1 61 40 00 00 00 00 00 00 00 00 04 4C 86 
22:17:03.610 > Interrupt received
22:17:03.663 > RX Channel: 23 --> 95 71 60 56 25 71 60 56 25 01 00 01 01 C4 00 D9 00 D2 03 D1 03 B1 00 0C 38 CB C4 | -80 dBm
22:17:03.739 > Interrupt received
22:17:03.790 > RX Channel: 23 --> 95 71 60 56 25 71 60 56 25 02 00 0D 78 48 8C 55 87 D9 01 C5 00 D5 00 D5 03 C0 2A | -80 dBm
22:17:03.842 > Interrupt received
22:17:03.940 > RX Channel: 3 --> 95 71 60 56 25 71 60 56 25 03 03 C3 00 10 38 10 00 0D 6A A5 87 76 87 F5 09 1C 3A | -80 dBm
22:17:03.988 > Interrupt received
22:17:04.256 > RX Channel: 3 --> 95 71 60 56 25 71 60 56 25 84 13 87 0E 57 00 C0 00 9D 03 E7 01 3D 00 17 46 58 50 | -80 dBm
22:17:04.356 > RX Period End
22:17:04.356 > Success
22:17:08.489 > Fetch inverter: 116171605625
22:17:08.551 > TX RealTimeRunData Channel: 3 --> 15 71 60 56 25 80 18 48 04 80 0B 00 66 A1 61 45 00 00 00 00 00 00 00 00 54 73 EC 
22:17:09.083 > RX Period End
22:17:09.083 > All missing
22:17:09.083 > Nothing received, resend whole request
22:17:09.083 > TX RealTimeRunData Channel: 23 --> 15 71 60 56 25 80 18 48 04 80 0B 00 66 A1 61 45 00 00 00 00 00 00 00 00 54 73 EC 
22:17:09.640 > RX Period End
22:17:09.640 > All missing
22:17:09.640 > Nothing received, resend whole request
22:17:09.640 > TX RealTimeRunData Channel: 40 --> 15 71 60 56 25 80 18 48 04 80 0B 00 66 A1 61 45 00 00 00 00 00 00 00 00 54 73 EC 
22:17:09.690 > Interrupt received
22:17:09.778 > RX Channel: 23 --> 95 71 60 56 25 71 60 56 25 01 00 01 01 C4 00 D9 00 D2 03 D1 03 B1 00 0C 38 CC C3 | -80 dBm
22:17:09.839 > Interrupt received
22:17:09.905 > RX Channel: 75 --> 95 71 60 56 25 71 60 56 25 02 00 0D 78 49 8C 56 87 DA 01 C5 00 D5 00 D5 03 C0 2B | -80 dBm
22:17:09.989 > Interrupt received
22:17:10.083 > RX Channel: 75 --> 95 71 60 56 25 71 60 56 25 03 03 C3 00 10 38 11 00 0D 6A A5 87 77 87 F5 09 1C 3A | -80 dBm
22:17:10.134 > Interrupt received
22:17:10.236 > RX Channel: 75 --> 95 71 60 56 25 71 60 56 25 84 13 87 0E 58 00 C0 00 9D 03 E7 01 3D 00 17 A0 D4 35 | -80 dBm
22:17:10.322 > RX Period End
22:17:10.322 > Success
22:17:13.490 > Fetch inverter: 116171605625
22:17:13.565 > TX RealTimeRunData Channel: 61 --> 15 71 60 56 25 80 18 48 04 80 0B 00 66 A1 61 4A 00 00 00 00 00 00 00 00 A4 32 52 
22:17:13.619 > Interrupt received
22:17:13.674 > RX Channel: 23 --> 95 71 60 56 25 71 60 56 25 01 00 01 01 C4 00 D9 00 D2 03 D1 03 B1 00 0C 38 CC C3 | -80 dBm
22:17:13.770 > Interrupt received
22:17:13.823 > RX Channel: 23 --> 95 71 60 56 25 71 60 56 25 02 00 0D 78 49 8C 56 87 DA 01 C5 00 D5 00 D5 03 C0 2B | -80 dBm
22:17:13.873 > Interrupt received
22:17:13.980 > RX Channel: 3 --> 95 71 60 56 25 71 60 56 25 03 03 C3 00 10 38 11 00 0D 6A A5 87 77 87 F5 09 1B 3D | -80 dBm
22:17:14.077 > Interrupt received
22:17:14.124 > RX Channel: 3 --> 95 71 60 56 25 71 60 56 25 84 13 87 0E 57 00 C2 00 9D 03 E7 01 3D 00 17 D6 BA 20 | -80 dBm
22:17:14.180 > RX Period End
22:17:14.180 > Success
22:17:18.496 > Fetch inverter: 116171605625
22:17:18.550 > TX RealTimeRunData Channel: 75 --> 15 71 60 56 25 80 18 48 04 80 0B 00 66 A1 61 4F 00 00 00 00 00 00 00 00 F4 0D 38 
22:17:18.601 > Interrupt received
22:17:18.650 > RX Channel: 40 --> 95 71 60 56 25 71 60 56 25 01 00 01 01 C4 00 D9 00 D2 03 D1 03 B1 00 0C 38 CC C3 | -80 dBm
22:17:18.705 > Interrupt received
22:17:18.788 > RX Channel: 23 --> 95 71 60 56 25 71 60 56 25 02 00 0D 78 49 8C 56 87 DA 01 C5 00 D5 00 D5 03 C0 2B | -80 dBm
22:17:18.838 > Interrupt received
22:17:18.996 > RX Channel: 23 --> 95 71 60 56 25 71 60 56 25 03 03 C3 00 10 38 11 00 0D 6A A5 87 77 87 F5 09 1C 3A | -80 dBm
22:17:19.053 > Interrupt received
22:17:19.105 > RX Channel: 3 --> 95 71 60 56 25 71 60 56 25 84 13 86 0E 58 00 C2 00 9D 03 E7 01 3D 00 17 41 CF CC | -80 dBm
22:17:19.197 > RX Period End
22:17:19.197 > Success
22:17:21.611 > TX ActivePowerControl Channel: 3 --> 51 71 60 56 25 80 18 48 04 81 0B 00 01 18 00 01 5B 41 6F 
22:17:21.660 > Interrupt received
22:17:21.712 > RX Channel: 40 --> D1 71 60 56 25 71 60 56 25 81 00 00 0B 00 14 07 48 | -80 dBm
22:17:23.624 > RX Period End
22:17:23.624 > Success
22:17:23.624 > Fetch inverter: 116171605625
22:17:23.711 > TX RealTimeRunData Channel: 23 --> 15 71 60 56 25 80 18 48 04 80 0B 00 66 A1 61 54 00 00 00 00 00 00 00 00 04 B3 6D 
22:17:24.215 > RX Period End
22:17:24.215 > All missing
22:17:24.215 > Nothing received, resend whole request
22:17:24.215 > TX RealTimeRunData Channel: 40 --> 15 71 60 56 25 80 18 48 04 80 0B 00 66 A1 61 54 00 00 00 00 00 00 00 00 04 B3 6D 
22:17:24.324 > Interrupt received
22:17:24.424 > RX Channel: 3 --> 95 71 60 56 25 71 60 56 25 01 00 01 01 C4 00 CE 00 C7 03 A0 03 82 00 0C 38 CC 83 | -80 dBm
22:17:24.496 > Interrupt received
22:17:24.550 > RX Channel: 75 --> 95 71 60 56 25 71 60 56 25 02 00 0D 78 49 8C 56 87 DA 01 C5 00 C9 00 CA 03 8C 64 | -80 dBm
22:17:24.600 > Interrupt received
22:17:24.657 > RX Channel: 75 --> 95 71 60 56 25 71 60 56 25 03 03 8F 00 10 38 11 00 0D 6A A5 87 77 87 F5 09 1A 70 | -80 dBm
22:17:24.712 > Interrupt received
22:17:24.761 > RX Channel: 61 --> 95 71 60 56 25 71 60 56 25 84 13 86 0D 99 00 C2 00 95 03 E6 01 3D 00 17 A8 D5 F4 | -80 dBm
22:17:24.826 > RX Period End
22:17:24.826 > Success

**Reply by stefan123t** [reply]
You can see that in the error state you are sending TX ActivePowerControl very frequently. It should not be sent more than every 15 seconds.
If you decrease the frequency of the control loop in reserve85 script it should work fine again.
You may also consider to try PM Power Meter + Dynamic Power Limiter options in the OpenDTU-OnBattery Fork which does the control on the OpenDTU without external script.
I have described a [possible PM+DPL setup](https://github.com/tbnobody/OpenDTU/issues/272#issuecomment-2219644352) with a Shelly Pro3EM as **Power Meter (PM)** here in issue #272.

**Reply by robson1311** [reply]
oh thanks, I'll try it and report back.

edit:
# interval time for setting limit to Hoymiles
LOOP_INTERVAL_IN_SECONDS = 10 
Set to 10 on 15, opendtu went offline after a few hours.
